content/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html

```diff
@@ -15,20 +15,20 @@
 
 <div class="layout" id="top">
 
-    <main class="content katacoda-content">
-
-        <div class="katacoda">
-            <div class="katacoda__alert">
-                टर्मिनल के साथ इंटरैक्ट करने के लिए स्क्रीन बहुत संकरी है, कृपया डेस्कटॉप या टैबलेट का उपयोग करें।
-            </div>
-            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/1" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;"></div>
-        </div>
-        <div class="row">
-            <div class="col-md-12">
-            	<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">होम<span class=""></span></a>
-                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">मॉड्यूल 2 जारी रखें &gt;<span class=""></span></a>
-            </div>
+    <main class="content">
+
+        <div>
+            <h2 class="katacoda-gone">
+                प्रलेखन अनुपलब्ध
+            </h2>
+            <p>
+                क्लस्टर बनाने के तरीके पर इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
+
+                अधिक जानकारी के लिए देखें <a href="https://kubernetes.io/pt-br/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
```

Comment:
This should redirect to "https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/"